### DISCUSSION TITLE
Building Custom Modules

### DISCUSSION THREAD

**Discussion by defencedog** [Joshua-Riek/ubuntu-rockchip#1062] [General]
Joshua is doing a splendid work, however, in his releases shoulld he add header `.debs` as well. I believe between cost of space & ease of use Joshua might want to build `.img` with _kernel headers_ included.

Secondly how do I install headers any way? For instance I am at 6.1Kernel so header files are where?

**Comment by hakotani-o** [comment]
Hi,defencedog.

Please check /usr/include.

Thank you.
